src/main/java/de/tum/in/www1/artemis/service/UserService.java

```diff
@@ -392,4 +392,13 @@ public User updateUserNotificationReadDate() {
         return loggedInUser;
     }
 
+    @Transactional(readOnly = true)
+    public User getUserById(Long tutorId) {
```

Comment:
Can you not use getUserWithAuthorities?
Definitely make sure tutorId --> userId

Reply:
You are right, I overthought this one. I removed the method and simplified the code where it was used